docs/src/content/mapping/ramda/pluck.md

```diff
@@ -0,0 +1,36 @@
+---
+category: List
+remeda: map
+---
+
+_Not provided by Remeda._
+
+- `pluck(k, data)` is equivalent to `R.map(R.prop(k), data)`. `map` and `prop`
```

Comment:
**question:** did you mean `R.map(data, R.prop(k))`?

Reply:
This is from the ramda docs, they are only data-last